nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_recovery.xml

```diff
@@ -20,12 +20,15 @@
       </PipelineSequence>
```

Comment:
We should update the https://github.com/ros-planning/navigation2/tree/main/nav2_bt_navigator readme for this file with a new image + description of the reactive recoveries

Reply:
And update the:
- migration guide on the website for these changes explaning them (recovery servers doing what is most logical as stand-alone agents, BT updated to allow for re-navigation between recoveries in the main branch, etc)

Reply:
I'm pretty sure I already asked if we should update any other BTs but I think the answer was 'no', but just checking again to verify. 

How does [navigate_w_replanning_and_round_robin_recovery.xml](https://github.com/ros-planning/navigation2/blob/dd34c895f784a9ac832111caef9a2861b7d9d2a2/nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_round_robin_recovery.xml)  compare? Should it be removed superseded by this or something we should add from this to make this overall supersede? At first glance, the major difference is `RoundRobin` in the navigation subtree recoveries & goal updated checks. Is that worth adding here and just removing that other tree?

Reply:
I think [navigate_w_replanning_and_round_robin_recovery.xml](https://github.com/ros-planning/navigation2/blob/dd34c895f784a9ac832111caef9a2861b7d9d2a2/nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_round_robin_recovery.xml) can be removed altogether.

The idea behind having it in the first place was to show how the RoundRobin node could be used to handle recoveries. If we're handling recoveries using RoundRobin in the default nav2 BT, there's no point keeping the other one.

Reply:
You sure there aren't elements of that one worth absorbing here? It looks like it might be more responsive to goal updates.

Reply:
please see https://github.com/ros-planning/navigation.ros.org/pull/136 for migration guide changes.